plugins/SnapshotPlugin/README.md

```diff
@@ -2,9 +2,9 @@
 
 > Generates V8 snapshot blobs of Webpack chunks at build time.
 
-Thanks to javascript virtual DOM technologies, it's possible to embed whole apps within V8 snapshot blobs for fast loading within tools like Chromuim and Electron. This is a highly-specific operation and definitely isn't ideal for everybody. However it is useful in certain scenarios and this will help simplify the process. The `SnapshotPlugin` assumes usage of React (and optionally Enact), with special handling to ensure APIs load correctly.
+Thanks to javascript virtual DOM technologies, it's possible to embed whole apps within V8 snapshot blobs for fast loading within tools like Chromium and Electron. This is a highly-specific operation and definitely isn't ideal for everybody. However it is useful in certain scenarios and this will help simplify the process. The `SnapshotPlugin` assumes usage of React (and optionally Enact), with special handling to ensure APIs load correctly.
```

Comment:
```suggestion
Thanks to javascript virtual DOM technologies, it's possible to embed whole apps within V8 snapshot blobs for fast loading within tools like Chromium and Electron. This is a highly-specific operation and definitely isn't ideal for everybody. However, it is useful in certain scenarios and this will help simplify the process. The `SnapshotPlugin` assumes usage of React (and optionally Enact), with special handling to ensure APIs load correctly.
```